packages/desktop-client/src/components/schedules/EditSchedule.js

```diff
@@ -122,7 +124,9 @@ export default function ScheduleDetails() {
         }
         case 'set-field':
           if (!(action.field in state.fields)) {
-            throw new Error('Unknown field: ' + action.field);
+            throw new Error(
+              t('schedules.unknownField', { field: action.field })
```

Comment:
I think we should leave these in hardcoded english tbh. They're not user facing errors afaik (I get the fatal error screen if I trigger this) so it just creates a layer of indirection when debugging.

Reply:
Done!